x-pack/plugins/siem/public/network/components/embeddables/map_tool_tip/point_tool_tip_content.test.tsx

```diff
@@ -46,32 +41,6 @@ describe('PointToolTipContent', () => {
     expect(wrapper.find('PointToolTipContentComponent')).toMatchSnapshot();
   });
 
-  test('renders array filter correctly', () => {
-    const closeTooltip = jest.fn();
-
-    const wrapper = mount(
-      <TestProviders>
-        <PointToolTipContentComponent
-          contextId={'contextId'}
-          featureProps={mockFeaturePropsArrayValue}
-          closeTooltip={closeTooltip}
-        />
-      </TestProviders>
-    );
-    expect(wrapper.find('[data-test-subj="add-to-kql-host.name"]').prop('filter')).toEqual({
-      meta: {
-        alias: null,
-        disabled: false,
-        key: 'host.name',
-        negate: false,
-        params: { query: 'testPropValue1' },
-        type: 'phrase',
-        value: 'testPropValue1',
-      },
-      query: { match: { 'host.name': { query: 'testPropValue1', type: 'phrase' } } },
-    });
-  });
-
```

Comment:
This test is irrelevant since we removed `AddFilterToGlobalSearchBar` and now have a [standardized tooltip](https://github.com/elastic/kibana/pull/61207) for draggables.